dist/torque.uncompressed.js

```diff
@@ -1635,6 +1635,8 @@ function GMapsTorqueLayer(options) {
 
   this.hidden = !this.options.visible;
 
+  this.showLimitErrors = options.showLimitErrors;
```

Comment:
This line seems a bit weird as we're not using it anywhere 🤔

Reply:
Oops! Bad copy/paste I guess  😱

Reply:
Humm, it comes from a 14 months ago commit, blame: @rubenmoya 😂 

Not sure if we should remove it. 

Have in mind that `dist` files are being regenerated like a year ago.

Reply:
I'm going to leave it as it is. If it's a bug, let's fix it in another PR

Reply:
Yes, let's leave it as is then 😁

Reply:
Hi! That comes from the errors overlay we tried to add a while back, seems like we removed it from every place but torque 😂 my bad!

Reply:
Too late 😂